changelogs/unreleased/gh-4317-console-add-line-carrying-backslash.md

```diff
@@ -0,0 +1,4 @@
+## feature/console
+
+* Now multiline commands can be used by setting up continuation marker
+  with `\set continuation` command.
```

Comment:
There is still the old alias in the commit message: `\set c off`.

Reply:
Fixed.